### PR TITLE
fix: project date

### PR DESCRIPTION
**Pull request by GugaGongadze**
## Description of change
This PR works with https://github.com/shuttle-hq/console-ui/pull/44
Appends project ID to the response payload so that its creation date can be deduced.

## How has this been tested? (if applicable)
Yes, tested in combination with Console UI

### CODE REVIEW

**Review by iulianbarbu — COMMENTED** [review]
Just a few improvements, otherwise looks good  👍

**Review by iulianbarbu — COMMENTED** [review]
LGTM! We'll only need to use `get_project` instead of `find_project_id` to squeeze two db calls into one.

File: gateway/src/api/latest.rs [hunk 1]
```diff
@@ -145,8 +146,9 @@ async fn get_projects_list(
         .iter_user_projects_detailed(&name, limit * page, limit)
         .await?
         .map(|project| project::Response {
-            name: project.0.to_string(),
-            state: project.1.into(),
+            id: project.0.to_string().to_uppercase(),
+            name: project.1.to_string(),
+            state: project.2.into(),
```
Comment (iulianbarbu) [inline gateway/src/api/latest.rs:151]: @GugaGongadze can we actually create structs that wrap around the info that gets sent around as a tuple? I am finding it difficult to read it based on tuple indexing. The structs can be defined in `service.rs` I think, but LMK what you think. This isn't necessarily blocking this PR either.

I see some tests are failing too. Can you look into them?

**Review by oddgrd — APPROVED** [review]
Looks great, thank you! :partying_face:

File: gateway/src/api/latest.rs (hunk 1 above)
Comment (iulianbarbu) [reply gateway/src/api/latest.rs:151]: nit: I am fine with leaving this as is for this PR, but something similar to `FindProjectPayload` would be needed here as well.

Resolving this.

**Review by iulianbarbu — APPROVED** [review]
LGTM! 👍

**Pull request merged**